docs/config/index.md

```diff
@@ -744,6 +744,14 @@ export default defineConfig({
 
   When set to `true`, the build will also generate a SSR manifest for determining style links and asset preload directives in production.
 
+### build.ssr
+
+- **Type:** `string`
```

Comment:
It accepts string or boolean as per https://github.com/vitejs/vite/blob/f6d08c7430906d6262ee1df729d2ea193a80fa6c/packages/vite/src/node/build.ts#L190

Reply:
ok, thanks

Reply:
has updated